### ISSUE TITLE
[Feature]: Support --version to show information about mo

### ISSUE THREAD

**Issue by m-schen** [matrixorigin/matrixone#1326]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```
mo-server --version

### Actual Behavior

mo-server --version not supported.

### Expected Behavior

need to supportmo-server --version to echo some information
"
go version : 
branch-name : 
last-commit-id :
build time :
mo version: v0.1 / v0.2
"

**Comment by broccoliSpicy** [comment]
This issue is solved by commit #1331. Pass the argument "--version" to mo-server executable and you will get the mo-server build information.

**Comment by lignay** [comment]
./mo-server --version
MatrixOne build info:
The golang version used to build this binary is: go version go1.17 darwin/arm64
The git branch name is: main
Last git commit ID: 3f32c36753ec9578e17333d395635a349ada1bb7
The Buildtime is: Thu Dec 16 20:39:01 CST 2021
Current Matrixone version: v0.1.0
